mmv1/products/clouddeploy/api.yaml

```diff
@@ -0,0 +1,234 @@
+                  input: true
+                  description: |
+                    Skaffold profiles to use when rendering the manifest for this stage's Target.
+  - !ruby/object:Api::Resource
+    name: 'Target'
+    base_url: projects/{{project}}/locations/{{region}}/targets
+    self_link: 'projects/{{project}}/locations/{{region}}/targets/{{name}}'
+    create_url: projects/{{project}}/locations/{{region}}/targets?targetId={{name}}
+    input: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/deploy/docs'
+      api: 'https://cloud.google.com/deploy/docs/api/reference/rest/v1/projects.locations.targets'
+    description: |-
+      A Target defines a location to which a Skaffold configuration can be deployed.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: region
+        description: The region of the Target Store. eg us-central1
+        url_param_only: true
+        input: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: The name of the Target. This value may be up to 60 characters, and valid characters are [a-z0-9_]. The first character cannot be a number.
+        input: true
+        url_param_only: true
+        pattern: projects/{{project}}/locations/{{region}}/targets/{{name}}
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: Description of the Target.
+        input: true
+      - !ruby/object:Api::Type::String
+        name: 'createTime'
+        output: true
+        description: |
+          The timestamp of when the Target was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::String
+        name: 'updateTime'
+        output: true
+        description: |
+          The timestamp of when the Target was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::String
+        name: 'etag'
+        output: true
+        description: |
+          This checksum is computed by the server based on the value of other fields, and may be sent on update and delete requests to ensure the client has an up-to-date value before proceeding.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          Resource labels to represent user-provided metadata.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'annotations'
+        description: |
+          User annotations. These attributes can only be set and used by the user, and not by Google Cloud Deploy.
+      - !ruby/object:Api::Type::Boolean
+        name: 'requireApproval'
+        description: |
+          Whether or not the Target requires approval.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'gke'
```

Comment:
Can we add an update test for this as well?

Reply:
Done